code/modules/mob/living/carbon/human/species.dm

```diff
@@ -981,6 +981,7 @@ GLOBAL_LIST_EMPTY(features_by_species)
 					if(!disable_warning)
 						to_chat(H, span_warning("[I] doesn't fit your [chest.name]!"))
 					return FALSE
+				return FALSE
```

Comment:
unmodular edit needs tagged and explained

Reply:
This part from my another branch.. miss this when marged. I remove this cnange in the next commit.